pkg/controller/master/backup/restore.go

```diff
@@ -220,11 +273,117 @@ func (h *RestoreHandler) PersistentVolumeClaimOnChange(key string, pvc *corev1.P
 		return nil, nil
 	}
 
+	volumeName, err := getVolumeName(pvc)
```

Comment:
No, because during the engine restoring phase, the pvc is not bound to volume yet, the `pvc.Spec.VolumeName` is null. We only can borrow the naming volume/pv naming convention from `externel-provisioner` and `LH`, thanks.